scenario/build/generate.xml

```diff
@@ -23,6 +23,7 @@
         </arguments>
     </step>
     <step name="refresh-modules" type="Magento\MagentoCloud\Step\Build\RefreshModules" priority="400"/>
+    <step name="copy-pub-static" type="Magento\MagentoCloud\Command\CopyPubStatic" priority="450"/>
```

Comment:
Now we can change here priority to 500 and add +100 to the next steps, as we didn't release yet.

Reply:
Fixed